hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieMergeOnReadRDD.scala

```diff
@@ -82,7 +82,8 @@ class HoodieMergeOnReadRDD(@transient sc: SparkContext,
                            @transient fileSplits: Seq[HoodieMergeOnReadFileSplit],
                            includeStartTime: Boolean = false,
                            startTimestamp: String = null,
-                           endTimestamp: String = null)
+                           endTimestamp: String = null,
+                           includedTimestamps: Set[String] = null)
```

Comment:
Are all these timestamps still representing instant times, not completion time?  The scaladocs should also be updated.

Reply:
Yes, these are still using instant times. Because they are used to filter records based on `HoodieRecord.COMMIT_TIME_METADATA_FIELD` in each record.

I'll fix the scaladoc.

Also, I think we can remove `startTimestamp` and `endTimestamp` since they are not used anymore, will file a jira to track this

Reply:
I've removed these unused variables in this PR